engine/proto/errorv2.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package enginepb;
+
+message ErrorV2 {
```

Comment:
Why use `V2`? Is there any compatible issues?

Reply:
There is `message Error` in `error.proto`.